src/flow-manager.c

```diff
@@ -566,6 +566,7 @@ static uint32_t FlowCleanupHash(void)
             FlowWakeupFlowRecyclerThread();
         }
     }
+    DEBUG_VALIDATE_BUG_ON(local_queue.len >= 25);
```

Comment:
Why 25 ?

Reply:
Because we already process them in chunks of 25 or above in the loop above so the items in the end must be less than 25 or else have already been processed in the loop.

Reply:
oh if your question was why is it processed in chunks of 25, idk that 😅